Controller overhault

V10lator opened the pull request:
Credits to Vague Rant for testing and the button map

V10lator committed:
Check VPADReadError

Signed-off-by: Thomas Rohloff <v10lator@myway.de>

V10lator committed:
Support more controllers

Signed-off-by: Thomas Rohloff <v10lator@myway.de>

V10lator committed:
Use emulated buttons for C

Signed-off-by: Thomas Rohloff <v10lator@myway.de>

V10lator committed:
Base button handling around the PSO N64 controller

Signed-off-by: Thomas Rohloff <v10lator@myway.de>

V10lator committed:
Base button mapping on design by Vague Rant

Signed-off-by: Thomas Rohloff <v10lator@myway.de>

vaguerant committed:
Make NSO layout optional, tweak Classic Controller

vaguerant commented:
Pull request-ception, I just made a PR to V10lator's fork which would affect this behavior. See V10lator#1 for details.

vaguerant committed:
Fix nonsense code

vaguerant committed:
boolify that bool

V10lator committed:
Merge pull request #1 from vaguerant/patch-1

Make NSO layout optional, tweak Classic Controller

V10lator committed:
Add rumble

Signed-off-by: Thomas Rohloff <v10lator@myway.de>

V10lator committed:
120 bit = 15 byte

Signed-off-by: Thomas Rohloff <v10lator@myway.de>

V10lator committed:
More simple rumble implementation

This one assumes that `controller->rumble` is always either 0 or 1 and that `VPADControlMotor` with a length of 0 does the same as `VPADStopMotor`.

Signed-off-by: Thomas Rohloff <v10lator@myway.de>

V10lator committed:
Update to newer WUT

Signed-off-by: Thomas Rohloff <v10lator@myway.de>
